### PR TITLE
Closure

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
[SP] Move reconstruction sample code to closure.

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@leonhsl please help to review. Thanks!

File: sample/src_polymer/elements/scene-perception-pages/reconstruction-page/reconstruction-page.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
+if (typeof spReconstructionSample == 'undefined')
+  spReconstructionSample = {};
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/reconstruction-page/reconstruction-page.js]: Where is `spReconstructionSample` defined?

**Comment by DonnaWuDongxia** [comment]
@leonhsl thanks for review. updated.

**Comment by DonnaWuDongxia** [comment]
@leonhsl updated and checked with sample installer. Thanks for reminding, the old version indeed introduce some issues on common functions reference. I have moved them to utils.js.

**Comment by leonhsl** [comment]
lgtm and merging..

**Comment by leonhsl** [comment]
Merged

**Pull request merged**